dev/tests/api-functional/testsuite/Magento/GraphQl/GiftMessage/Cart/GiftMessageTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\GiftMessage\Cart;
+
+use Exception;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\GraphQl\Quote\GetMaskedQuoteIdByReservedOrderId;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+
+class GiftMessageTest extends GraphQlAbstract
```

Comment:
Could you add a negative test case? Where you attempt to retrieve gift message for a cart that has none.

Reply:
**Done**